pkg/isoeditor/initramfs.go

```diff
@@ -0,0 +1,27 @@
+package isoeditor
+
+import (
+	"io"
+	"os"
+
+	"github.com/openshift/assisted-image-service/pkg/overlay"
+	"github.com/pkg/errors"
+)
+
+func NewInitRamFSStreamReader(irfsPath string, ignitionContent *IgnitionContent) (io.ReadSeeker, error) {
```

Comment:
Is this function unused on purpose?

Reply:
This is a new public api.
@zaneb isn't using this as a service, but as a library.

Reply:
Ack